src/features/features.pug

```diff
@@ -37,14 +37,14 @@ section.feature.is-alt
         pre.code
           :lowlight( lang="css" prefix="code-" )
             body {
-              color: lch(53 105 40);
+              color: oklch(53% 105 40);
```

Comment:
105 is illegal value for `C` component (it is mostly between 0 and 0.4)

I fixed it here cbc9327